libr/core/cmd_print.c

```diff
@@ -4354,13 +4355,77 @@ static const char* bits_to_c_code_fmtstr(int bits) {
 	}
 }
 
+static void print_c_instructions(RPrint *p, ut64 addr, const ut8 *buf, int len, int ws) {
+	size_t i;
+
+	ws = R_MAX (1, R_MIN (ws, 8));
+	int bits = ws * 8;
+	const char *fmtstr = bits_to_c_code_fmtstr (bits);
+	len /= ws;
+
+	p->cb_printf ("#define _BUFFER_SIZE %d\n", len);
+	p->cb_printf ("const uint%d_t buffer[_BUFFER_SIZE] = {\n", bits);
+	p->interrupt = false;
```

Comment:
interrupt is never set to true in the entire codebase AFAIU, so this can be completely remove (you can start by just not using it in your function I guess)

Reply:
it is, done by rcons_break_begin() and so on. its how rprint handle user interrupt events

Reply:
where? I can't find it, could you point me to the exact line where `p->interrupt` is set to true?

Reply:
oh, you are right, its just using a callback, fixed. thanks!